reco_utils/recommender/vae/multinomial_vae.py

```diff
@@ -266,223 +259,241 @@ def __init__(
-                                  mean=0.0, 
-                                  stddev = 1.0,
-                                  seed=self.seed
-                                  )
-            
-        return _mean + K.exp(_log_var / 2) * epsilon
+        # epsilon
+        epsilon = K.random_normal(
+            shape=(K.shape(_mean)[0], self.latent_dim),
+            mean=0.0,
+            stddev=1.0,
+            seed=self.seed,
+        )
 
+        return _mean + K.exp(_log_var / 2) * epsilon
 
     def nn_batch_generator(self, x_train):
         """Used for splitting dataset in batches.
-        
-        Args: 
-            x_train (np.array, float): the click matrix for the train set.
+
+        Args:
+            x_train (np.array): The click matrix for the train set, with float values.
         """
         # Shuffle the batch
-        np.random.seed(self.seed) 
+        np.random.seed(self.seed)
         shuffle_index = np.arange(np.shape(x_train)[0])
         np.random.shuffle(shuffle_index)
         x = x_train[shuffle_index, :]
         y = x_train[shuffle_index, :]
-                
+
         # Iterate until making a full epoch
         counter = 0
         while 1:
-            index_batch = shuffle_index[self.batch_size * counter : self.batch_size * (counter + 1)]
-            # Decompress batch 
-            x_batch = x[index_batch,:] 
-            y_batch = y[index_batch,:]
+            index_batch = shuffle_index[
+                self.batch_size * counter : self.batch_size * (counter + 1)
+            ]
+            # Decompress batch
+            x_batch = x[index_batch, :]
+            y_batch = y[index_batch, :]
             counter += 1
             yield (np.array(x_batch), np.array(y_batch))
-            
+
             # Stopping rule
             if counter >= self.number_of_batches:
-                counter = 0 
+                counter = 0
 
     def fit(self, x_train, x_valid, x_val_tr, x_val_te, mapper):
         """Fit model with the train sets and validate on the validation set.
-            
-        Args: 
-            x_train (np.array, float): the click matrix for the train set.
-            x_valid (np.array, float): the click matrix for the validation set.
-            x_val_tr (np.array, float): the click matrix for the validation set training part.
-            x_val_te (np.array, float): the click matrix for the validation set testing part.
-            mapper (AffinityMatrix): the mapper for converting click matrix to dataframe.
+
+        Args:
+            x_train (np.array): the click matrix for the train set.
+            x_valid (np.array): the click matrix for the validation set.
+            x_val_tr (np.array): the click matrix for the validation set training part.
+            x_val_te (np.array): the click matrix for the validation set testing part.
+            mapper (obj): the mapper for converting click matrix to dataframe. It can be AffinityMatrix.
```

Comment:
`numpy.ndarray` 
Do we know what the object is? Otherwise it should be `object`.

Reply:
all "obj" is changed to "object"